src/pages/es/guides/images.md

```diff
@@ -113,58 +113,54 @@ Consulta la [guía de la integración image](/es/guides/integrations-guide/image
+Para usar una imagen remota, pasa una URL completa en el atributo `src` de `<Image />`. `<Image />` no inferirá las dimensiones de este archivo remoto. Debes proporcionar el `formato` para renderizar el width de la imagen, y debes proporcionar `width` y `height` o uno de los dos tamaños junto a `aspectRatio`. El atributo `alt` también es requerido.
 
 #### Ejemplos
 
 ```astro
 ---
 // src/pages/index.astro
 import { Image } from '@astrojs/image/components';
-import localImage from '../assets/local.png';
-const imageUrl = 'https://astro.build/assets/logo.png';
+import localImage from '../assets/logo.png';
+const remoteImage = 'https://picsum.photos/id/957/300/200.jpg';
+const localAlt = 'El Logo de Astro';
+const remoteAlt = 'Una vista de un bosque durante el día';
 ---
 
-// Imagen local optimizada, manteniendo el ancho, alto y formato de imagen original
-<Image src={localImage} />
+<!-- Imagen local optimizada, manteniendo el ancho, alto y formato de imagen original -->
+<Image src={localImage} alt={localAlt} />
+
+<!-- height se recalculará para que coincida con la relación de aspecto original -->
+<Image src={localImage} width={300} alt={localAlt} />
 
-// La altura se recalculara para que coincida con la original (solo local) o con la relación de aspecto especificada
-<Image src={localImage} width={300} />
-<Image src={imageUrl} width={300} aspectRatio={16/9} />
+<!-- Para imágenes remotas, las dimensiones deseadas y el formato son requeridos -->
+<Image src={remoteImage} width={300} aspectRatio="1:1" format="png" alt={remoteAlt} />
 
-// Ajustar a un ancho y alto específicos
-<Image src={localImage} width={300} height={600} />
-<Image src={imageUrl} width={544} height={184} />
+<!-- Ajustar a un ancho y alto específicos -->
+<Image src={localImage} width={300} height={600} alt={localAlt}/>
+<Image src={remoteImage} width={544} height={184} format="png" alt={remoteAlt}/>
 
-// Ajustar a una relación de aspecto específica y convertir a formato avif 
-<Image src={localImage} aspectRatio="16:9" format="avif" />
-<Image src={imageUrl} height={200} aspectRatio="16:9" format="avif" />
+<!-- Ajustar a una relación de aspecto específica y convertir a formato avif -->
+<Image src={localImage} aspectRatio="16:9" format="avif" alt={localAlt}/>
+<Image src={remoteImage} height={200} aspectRatio="16:9" format="avif" alt={remoteAlt}/>
 
-// Las imágenes locales también pueden importarse directamente
-<Image src={import('../assets/local.png')} />
+<!-- Importaciones de imágenes locales pueden hacerse en línea -->
+<Image src={import('../assets/logo.png')} alt={localAlt}/>
+<!-- Si una imagen es alojada en el directorio `/public`, usa una ruta relativa a `/public` -->
+<Image src="/penguin.jpg" width="300" aspectRatio={1} format="png" alt="A happy penguin"/>
```

Comment:
Missing some spacing between the final attributes and the self-closing tag. E.g.

```jsx
// This
<Image src={localImage} ... alt={localAlt}/>

// Should be this
<Image src={localImage} ... alt={localAlt} />
```